### PR TITLE
feat/submit mentee talks

### PR DESCRIPTION
**Pull request by KenanHaciyev**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -23,6 +21,14 @@ const routes: Routes = [
...
 				(m) => m.TalksPageModule
 			)
+	},
+	{
+		path: "mentee-page",
+		loadChildren: () =>
+			import("../../projects/core/src/lib/components/mentee-page/mentee-page.module").then(
+				(m) => m.MenteePageModule
+			),
+		canActivate: [AuthGuard]
```
Comment (Tsurhanau) [inline src/app/app-routing.module.ts:31]: AuthGuard is not need here

File: projects/common-components/src/lib/talks/talk-item/talk-item.component.spec.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // import { ComponentFixture, TestBed } from "@angular/core/testing";
-// import { TalkItemComponent } from "@jsmu/common-components";
+// import { TalkItemComponent } from "./talk-item.component";
 // import { talkMock } from "../../../../../../src/assets/tests/talkMocks";
 // import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
 //
```
Comment (Tsurhanau) [inline projects/common-components/src/lib/talks/talk-item/talk-item.component.spec.ts:5]: uncomment

Comment (KenanHaciyev) [reply projects/common-components/src/lib/talks/talk-item/talk-item.component.spec.ts]: done

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (KenanHaciyev) [reply src/app/app-routing.module.ts]: done

**Review by Tsurhanau — APPROVED** [review]

**Pull request merged**